tests/common/devices/sonic.py

```diff
@@ -185,6 +185,16 @@ def _gather_facts(self):
         logging.debug("Gathered SonicHost facts: %s" % json.dumps(facts))
         return facts
 
+    def _get_namespace_for_port(self, port_name):
```

Comment:
can we reuse this function https://github.com/Azure/sonic-mgmt/blob/003a37cf6315eb044722bfaf5ad58e8ad9992152/tests/common/devices/multi_asic.py#L307

Reply:
Thank you, I have modified as suggested.